iggy-enrich-demo/iggy_enrich_demo.ipynb

```diff
@@ -0,0 +1,791 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from google.cloud import storage\n",
+    "\n",
+    "def download_public_file(bucket_name, prefix, desired_blobs):\n",
+    "    \n",
+    "    storage_client = storage.Client.create_anonymous_client()\n",
+    "    bucket = storage_client.bucket(bucket_name)\n",
+    "\n",
+    "    blobs = bucket.list_blobs(prefix=prefix)\n",
+    "    for blob in blobs:\n",
+    "        if blob.name in desired_blobs:\n",
+    "            blob.download_to_filename(blob.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40488203-fb27-475e-b9b4-cf98cc05ad1b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!mkdir austin-datasets\n",
+    "download_public_file('iggy-web-demo', 'austin-datasets', ['austin-datasets/samples_iggy-package-wkt-20211209155137_tx_austin_quadkeys.tar.gz', 'austin-datasets/vacation_rentals.tar.gz'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ba2e8e76",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!tar xzvf austin-datasets/samples_iggy-package-wkt-20211209155137_tx_austin_quadkeys.tar.gz -C austin-datasets/\n",
+    "!tar xzvf austin-datasets/vacation_rentals.tar.gz -C austin-datasets/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32971e28-12aa-43a0-a3b3-e3deacff267a",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Load 3rd Party Data and Enrich with Iggy Data\n",
+    "Now we will load some third party data - information on vacation rentals in Austin, TX and enrich this data with Iggy data. We will use a selected subset of features spanning the zipcode, census block group (CBG) and Isochrone 10 minute walk boundaries. For more information on our data and the boundaries please read our [Data Readme](https://docs.askiggy.com/reference/place-data).\n",
+    "\n",
+    "We will enrich the vacation listings data using the [IggyEnrich](https://pypi.org/project/iggyenrich/0.0.2/) Python package.\n",
+    "This package makes it very easy to enrich your own data using selected features and to selected boundaries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3a9266b-b4f7-433f-813f-28a35c35a4aa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vacation_rental_data = pd.read_csv('austin-datasets/vacation_rental_austin.csv')\n",
+    "rental_data_clean = vacation_rental_data.copy()\n",
+    "rental_data_clean.drop(['name'], axis=1, inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "524dded6-6cc1-4bd5-bfef-e96488bbf86a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vacation_rental_columns = list(rental_data_clean.columns)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c96d084e-2c27-4a6a-94e5-c47f54e8af0e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "features = [\n",
```

Comment:
Include comment here to the effect of "here we are selecting particular Iggy features to enrich our dataset with"